engine/flink/management/src/it/scala/pl/touk/nussknacker/engine/management/streaming/StreamingDockerTest.scala

```diff
@@ -12,7 +13,8 @@ import pl.touk.nussknacker.engine.deployment.{DeploymentData, ExternalDeployment
 import pl.touk.nussknacker.engine.kafka.KafkaClient
 import pl.touk.nussknacker.engine.management.DockerTest
 
-trait StreamingDockerTest extends DockerTest with Matchers with OptionValues { self: Suite =>
+trait StreamingDockerTest extends DockerTest with BeforeAndAfterAll with Matchers with OptionValues {
```

Comment:
Why can't this extend LazyLogging instead of having the test classes extend it? But I see it in other previous cases so I'm pretty sure theres a reason

Reply:
It was just for sake of clarity. I don't like when I have multiple traits and each of them extends the same trait. IMO it is better to mix in as many things as possible on the top of hierarchy. Thanks to that the hierarchy is flat and without duplications.